contracts/wallet/BaseMultiSigWallet.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity 0.4.18;
 
 /// @title Multisignature wallet - Allows multiple parties to agree on transactions before execution.
 /// @author Stefan George - <stefan.george@consensys.net>
-contract MultiSigWallet {
+contract BaseMultiSigWallet {
```

Comment:
Should we make this abstract? Or a library -- basically anything that prevents it from being instantiable on its own.

Reply:
Let's table this for now.